src/storage/memory_store.ts

```diff
@@ -118,6 +119,11 @@ export class MemoryStore implements IStore {
    * where uid is the token uid in hex
    */
   registeredTokens: Map<string, ITokenData>;
+  /**
+   * Map<ncKey, INcData>
+   * where ncKey is a pair address:ncId concatenated.
```

Comment:
Old docstring, we are not using this key anymore, only `ncId`

Reply:
Fixed at https://github.com/HathorNetwork/hathor-wallet-lib/pull/614/commits/0a464ebc09a7adfa2686ed5b7496349539229341